### ISSUE TITLE
update default send and recv rates for p2p

### ISSUE THREAD

**Issue by ebuchman** [tendermint/tendermint#1752]
Currently set to 512000 bytes/sec.

Lets go 10x :)

**Comment by ValarDragon** [comment]
Do we want this done in tendermint? We've done this in the sdk (it intercepts the call to generate default config and updates via that)

**Comment by xla** [comment]
@ValarDragon Yes, but ideally we want to get rid of those alltogether, as per tendermint core discussion. We should communicate with peers as fast as possible and only start to throttle if we experience backpressure from a peer.

**Comment by vrde** [comment]
> @ValarDragon Yes, but ideally we want to get rid of those alltogether, as per tendermint core discussion.

Do you know when you'll get rid of this? One of the key features of Tendermint is to be _lightning fast_, 
but the default values for `send_rate` and `recv_rate` are going in the opposite direction.
